app/views/katello/api/v2/capsule_content/sync_status.json.rabl

```diff
@@ -18,15 +18,13 @@ child @lifecycle_environments => :lifecycle_environments do
     @capsule.environment_syncable?(env)
   end
 
-  if @capsule.has_feature?(SmartProxy::PULP_NODE_FEATURE)
+  if @capsule.has_feature?(SmartProxy::PULP_NODE_FEATURE) || @capsule.has_feature?(SmartProxy::PULP3_FEATURE)
     node :counts do |env|
-      counts = {
+      {
         :content_hosts => env.hosts.authorized("view_hosts").count,
         :content_views => env.content_views.non_default.count,
         :products => env.products.enabled.count
```

Comment:
i would probably drop the content_hosts and products api part as wel

Reply:
Updated to address comments..Also moved common methods to smart proxy..( Removed puppet env from pulp2 as well..I'm guessing that's ok?)